kind/kind-up-test.sh

```diff
@@ -0,0 +1,46 @@
+# run this from root of repository
+
+# spin up kind cluster
+cat <<EOF | sudo kind create cluster --image kindest/node:v1.28.0 --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+EOF
+
+sudo kind get kubeconfig > /tmp/config
+sudo chown $USER:$USER /tmp/config
+mv /tmp/config ~/.kube/config
```

Comment:
Should we do some conditional checking here if they have an existing `kubeconfig`, and if so add an entry for this kind cluster, or is this not an issue? Upon trying to debug the script, I realized that my existing quite extensive `kubeconfig` got wiped.

Reply:
oh shoot sorry Greg!! Definitely need that

Reply:
I updated the kind setup script to merge kubeconfigs rather than overwrite